app/views/citizens/income_summary/_income_type_item.html.erb

```diff
@@ -0,0 +1,16 @@
+<li>
+  <h2 class="app-task-list__section" id="<%= "income-type-#{name}" %>">
+    <span class="app-task-list__section-number"><%= number %>. </span>
+    <%= t("transaction_types.names.#{name}") %>
+    <% if I18n.exists? "transaction_types.hints.#{name}" %>
+      <span id="nationality-1-item-hint" class="govuk-hint" style="margin-bottom: 0px">
```

Comment:
what has `nationality` to do with transaction types?
Do we need an id on this `span` element?